BootloaderCorePkg/Stage1B/Stage1B.c

```diff
@@ -731,8 +730,8 @@ ContinueFunc (
                     MbTmpAlgHash,
                     Stage1bParam->ConfigDataHash,
                     EV_PLATFORM_CONFIG_FLAGS,
-                    sizeof(CfgDataFwBlob),
-                    (UINT8 *)&CfgDataFwBlob);
+                    (UINT32)CfgDataFwBlob.BlobLength,
```

Comment:
How about directly use CfgDataBlob->UsedLength and CfgDataFwBlob could be removed?

Reply:
done